app/scripts/lib/validate.js

```diff
@@ -103,7 +103,41 @@ define([
       }
 
       return password.length >= Constants.PASSWORD_MIN_LENGTH;
+    },
+
+    /**
+     * Check if a value is not falsy.
+     *
+     * @return the value if it's not falsy, the defaultValue otw.
+     */
+    isNotFalsy: function (value, defaultValue) {
+      return value || defaultValue;
+    },
+
+    /**
+     * Check if at least one of the arguments is not falsy.
+     *
+     * @return the first non falsy argument if there is one, the last argument
+     * otw.
+     */
+    areNotFalsy: function () {
```

Comment:
ok, i  have renamed them.
But I want to keep both of them as I have a feeling that the first function is faster then the second one